sfaira/data/dataloaders/loaders/d10_1038_s41586_018_0654_5/Mouse_Brain_2018_Smart-seq2_Tasic_001.py

```diff
@@ -0,0 +1,25 @@
+import anndata
+import os
+import scipy.sparse
+import numpy as np
+
+
+def load(data_dir, sample_fn, **kwargs):
```

Comment:
If you only have a single file, it is not necessary to use sample_fn and you can use `def load(data_dir, **kwargs):` instead here